src/AngleSharp.Diffing/Strategies/AttributeStrategies/OrderedStyleAttributeComparer.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Linq;
+
+using AngleSharp.Css.Dom;
+using AngleSharp.Diffing.Core;
+using AngleSharp.Dom;
+
+namespace AngleSharp.Diffing.Strategies.AttributeStrategies
+{
+    /// <summary>
+    /// Represents the style attribute comparer strategy.
+    /// </summary>
+    public static class OrderedStyleAttributeComparer
```

Comment:
I am regretting the word `Ordered` in the name of the type, because isn't this actually the opposite. We are explicitly ignoring the order of the style information in the style attribute, and NOT doing an ordered comparison.